python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py

```diff
@@ -255,6 +255,11 @@ def __from_edgelist(
             elif elist[source].dtype not in [np.int32, np.int64] or elist[
                 destination
             ].dtype not in [np.int32, np.int64]:
+                if elist[destination].dtype in [np.uint32, np.uint64]:
```

Comment:
should this also check `elist[source].dtype` ?

Reply:
fixed